test/blockwisealgorithms/CMakeLists.txt

```diff
@@ -1,4 +1,14 @@
 VIGRA_CONFIGURE_THREADING()
-VIGRA_ADD_TEST(test_blockwiselabeling test_labeling.cxx)
-VIGRA_ADD_TEST(test_blockwisewatersheds test_watersheds.cxx)
-VIGRA_ADD_TEST(test_blockwiseconvolution test_convolution.cxx)
+if(NOT THREADING_FOUND)
```

Comment:
I'm not sure, but probably not.  On most platforms, threading support is available.  My guess is that people with old compilers are probably more interested in bug fixes for the features they already use than in new features like the chunked array.